Dockerfile

```diff
@@ -9,8 +9,8 @@ WORKDIR /app
 RUN apt-get update && apt-get upgrade -y &&\
   apt-get install -y python3 python3-pip nano ffmpeg
 
+RUN python3 -m pip config set global.break-system-packages true
 COPY Pipfile Pipfile.lock ./
-RUN python3 -m pip install --upgrade pip
```

Comment:
This line will fail because the pip install is managed by apt. There is no need to update here if it is updated on line 9.